pyproject.toml

```diff
@@ -17,25 +17,22 @@ classifiers=[
     "Intended Audience :: Science/Research",
     "Development Status :: 5 - Production/Stable",
     "Programming Language :: Python",
-    "Programming Language :: Python :: 3.7",
-    "Programming Language :: Python :: 3.8",
-    "Programming Language :: Python :: 3.9",
```

Comment:
Hm, given that Python 3.9 lives until October 2025, I'm a bit uneasy with that, though I understand your reasoning. Ultimately, it is your decision.

Reply:
I'm sure Python 3.12 will be in official release by the time we get to this, likely for quite a while.  I understand your unease, but I think we have to take the opportunity to "break" things with the major version, and Python 3.10 adds a lot of value it would be very nice to take advantage of soon, rather than at v4.0.